control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -741,6 +741,53 @@ func (r *reconciler) reconcileCloudCredentialSecrets(ctx context.Context, hcp *h
 				errs = append(errs, fmt.Errorf("failed to reconcile aws cloud credential secret %s/%s: %w", secret.Namespace, secret.Name, err))
 			}
 		}
+	case hyperv1.AzurePlatform:
+		referenceCredentialsSecret := &corev1.Secret{ObjectMeta: metav1.ObjectMeta{Namespace: hcp.Namespace, Name: hcp.Spec.Platform.Azure.Credentials.Name}}
```

Comment:
After creating this change, it occurred to me that we could instead also just set up the cloud credentials operator on Azure. That would be a bit more futureproof, because it wouldn't require us to change code whenever a component that needs cloud credentials is added/removed to/from OCP. It would break the assumption though that we don't deploy it. WDYT @csrwng ?

Reply:
I don't think we want to get the cco involved if the general direction we want to go is to not mint creds. What you have here right now seems fine as a starting point. We can iterate on optionally refining the creds given to us.

Reply:
sgtm. Are we waiting for some more feedback or is this good to go?